app/routes/helpers.py

```diff
@@ -0,0 +1,140 @@
+from flask import jsonify, abort, make_response, request
+from dotenv import load_dotenv
+import requests
+from datetime import datetime
+from app import db
+import os
+
+load_dotenv()
+
+def send_slack_message(completed_task):
+    TOKEN = os.environ['SLACK_API_TOKEN']
```

Comment:
You'll work on APIs where a route will have to make more than one external call to an API so we should be more specific about the kind of token we're using. This variable could be called `SLACK_API_TOKEN`